docs/hello_nextflow/06_hello_config.md

```diff
@@ -259,71 +515,617 @@ NXF_CHDIR=/home/gitpod/work/34/850fe31af0eb62a0eb1643ed77b84f
+params {
+    // Primary input (file of input files, one per line)
+    reads_bam        = null
+
+    // Accessory files
+    reference        = null
+    reference_index  = null
+    reference_dict   = null
+    intervals        = null
+
+    // Base name for final output file
+    cohort_name      = "my_cohort"
+}
+```
+
+Now, if you run the same command again, it will still work.
+So yes, we're definitely able to pull those parameter values from the parameter file.
+
+This is great because, with the parameter file in hand, we'll now be able to provide parameter values at runtime without having to type massive command lines **and** without modifying the workflow nor the default configuration.
+
+That being said, it was nice to be able to demo the workflow without having to keep track of filenames and such. Let's see if we can use a profile to replicate that behavior.
+
+### 5.6. Create a demo profile
+
+Yes we can! We just need to retrieve the default parameter declarations as they were written in the original workflow (with the `params.*` syntax) and copy them into a new profile that we'll call `demo`.
+
+_Before:_
+
+```groovy title="nextflow.config" linenums="3"
+profiles {
+    docker_on {
+        docker.enabled = true
+    }
+    conda_on {
+        conda.enabled = true
+    }
+    my_laptop {
+        process.executor = 'local'
+        docker.enabled = true
+    }
+    univ_hpc {
+        process.executor = 'slurm'
+        conda.enabled = true
+        process.resourceLimits = [
+            memory: 750.GB,
+            cpus: 200,
+            time: 30.d
+        ]
+    }
+}
+```
+
+_After:_
+
+```groovy title="nextflow.config" linenums="3"
 profiles {
-    docker {
+    docker_on {
         docker.enabled = true
-        conda.enabled = false
     }
-    conda {
-        docker.enabled = false
+    conda_on {
         conda.enabled = true
     }
-    local {
+    my_laptop {
         process.executor = 'local'
+        docker.enabled = true
     }
-    slurm {
+    univ_hpc {
         process.executor = 'slurm'
+        conda.enabled = true
+        process.resourceLimits = [
+            memory: 750.GB,
+            cpus: 200,
+            time: 30.d
+        ]
+    }
+    demo {
+        // Primary input (file of input files, one per line)
+        params.reads_bam        = "data/sample_bams.txt"
+
+        // Accessory files
+        params.reference        = "data/ref/ref.fasta"
+        params.reference_index  = "data/ref/ref.fasta.fai"
+        params.reference_dict   = "data/ref/ref.dict"
+        params.intervals        = "data/ref/intervals.bed"
+
+        // Base name for final output file
+        params.cohort_name      = "family_trio"
     }
 }
 ```
 
-Now run the pipeline using two profiles, `docker` and `local`:
+As long as we distribute the data bundle with the workflow code, this will enable anyone to quickly try out the workflow without having to supply their own inputs or pointing to the parameter file.
```

Comment:
```suggestion
As long as we distribute the data bundle with the workflow code, this will enable anyone to quickly try out the workflow without having to supply their own inputs or pointing to the parameter file. Besides, we can provide URLs to where files are stored and Nextflow will download them automatically.
```